### PR TITLE
Implementation of a markdown document

### PR DESCRIPTION
**Pull request by sdellefratte**
### Description

I extracted the markdown.md document from the path docs/content/markdown and implemented it with some sentences taken from the codecademy.com course, to add a more complete description to the article.

### Type of Change

- Adding a new entry

### Checklist

- [ ] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Comment by sdellefratte** [comment]
Good afternoon, clicking on the  Contributor License Agreement<https://urldefense.proofpoint.com/v2/url?u=https-3A__cla-2Dassistant.io_Codecademy_docs-3FpullRequest-3D5281&d=DwMCaQ&c=cCoa5WWAB7EEETJScYfkXg&r=L1Mw0Abj985lMM4AtbOB-hPoMnBbdiy-l7OoxFES8bPGD8CUazRWxfNy1yL3LrUR&m=BmDMdNngHyPFXb0M-a6ooCV0AlT5-cuN0Nga2apspJyiC3AOISh6PZ8FiMbng78e&s=-2JDxrSRMmqcEYpgTzRQFOafpAWC_ZdDUEbAFtTdx0Q&e=> link, I see this screen, but I can't click or write anything.
Could you send me the right link? Thank you
Greetings


[cid:88812d45-7d3c-4169-9625-39e42d8d3c13]

Cordiali saluti
Stefania Delle Fratte
[cid:cc3619db-9c70-4955-a378-03eaaf80c7a0]
Kyndryl Senior Associate, Infrastructure Specialist
Via Luigi Stipa 150 - Roma
Mail: ***@***.******@***.***>
Cell.: +39 340 377 7648
________________________________
Da: CLAassistant ***@***.***>
Inviato: mercoledì 18 settembre 2024 16:59
A: Codecademy/docs ***@***.***>
Cc: STEFANIA DELLE FRATTE ***@***.***>; Author ***@***.***>
Oggetto: [EXTERNAL] Re: [Codecademy/docs] Implementation of a markdown document (PR #5281)

Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our Contributor License Agreement before we can accept your contribution. You have signed the CLA already but the status is still pending?


[CLA assistant check]<https://urldefense.proofpoint.com/v2/url?u=https-3A__cla-2Dassistant.io_Codecademy_docs-3FpullRequest-3D5281&d=DwMCaQ&c=cCoa5WWAB7EEETJScYfkXg&r=L1Mw0Abj985lMM4AtbOB-hPoMnBbdiy-l7OoxFES8bPGD8CUazRWxfNy1yL3LrUR&m=BmDMdNngHyPFXb0M-a6ooCV0AlT5-cuN0Nga2apspJyiC3AOISh6PZ8FiMbng78e&s=-2JDxrSRMmqcEYpgTzRQFOafpAWC_ZdDUEbAFtTdx0Q&e=>
Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our Contributor License Agreement<https://urldefense.proofpoint.com/v2/url?u=https-3A__cla-2Dassistant.io_Codecademy_docs-3FpullRequest-3D5281&d=DwMCaQ&c=cCoa5WWAB7EEETJScYfkXg&r=L1Mw0Abj985lMM4AtbOB-hPoMnBbdiy-l7OoxFES8bPGD8CUazRWxfNy1yL3LrUR&m=BmDMdNngHyPFXb0M-a6ooCV0AlT5-cuN0Nga2apspJyiC3AOISh6PZ8FiMbng78e&s=-2JDxrSRMmqcEYpgTzRQFOafpAWC_ZdDUEbAFtTdx0Q&e=> before we can accept your contribution.
You have signed the CLA already but the status is still pending? Let us recheck<https://urldefense.proofpoint.com/v2/url?u=https-3A__cla-2Dassistant.io_check_Codecademy_docs-3FpullRequest-3D5281&d=DwMCaQ&c=cCoa5WWAB7EEETJScYfkXg&r=L1Mw0Abj985lMM4AtbOB-hPoMnBbdiy-l7OoxFES8bPGD8CUazRWxfNy1yL3LrUR&m=BmDMdNngHyPFXb0M-a6ooCV0AlT5-cuN0Nga2apspJyiC3AOISh6PZ8FiMbng78e&s=O2lkfmzI4CWC5KeSFEyGQTb3P9TXkqL-B1jlXBz6Y8g&e=> it.

—
Reply to this email directly, view it on GitHub<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_Codecademy_docs_pull_5281-23issuecomment-2D2358715123&d=DwMCaQ&c=cCoa5WWAB7EEETJScYfkXg&r=L1Mw0Abj985lMM4AtbOB-hPoMnBbdiy-l7OoxFES8bPGD8CUazRWxfNy1yL3LrUR&m=BmDMdNngHyPFXb0M-a6ooCV0AlT5-cuN0Nga2apspJyiC3AOISh6PZ8FiMbng78e&s=83TWPDuz857fZpGdoKEOtrMmbilNU-2F3X-IgpeUZNE&e=>, or unsubscribe<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_BLLTG4MINORD623JLAMNX6DZXGINPAVCNFSM6AAAAABON4SHEGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDGNJYG4YTKMJSGM&d=DwMCaQ&c=cCoa5WWAB7EEETJScYfkXg&r=L1Mw0Abj985lMM4AtbOB-hPoMnBbdiy-l7OoxFES8bPGD8CUazRWxfNy1yL3LrUR&m=BmDMdNngHyPFXb0M-a6ooCV0AlT5-cuN0Nga2apspJyiC3AOISh6PZ8FiMbng78e&s=9SZAGKhvlLtyFraWQAikXgd0_cww7C6SJRF5fAp3P5A&e=>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

Unless stated otherwise above:
Kyndryl Italia S.p.A.
Società con unico azionista
Soggetta all'attività di direzione e coordinamento di Kyndryl, Inc.
Cap. Int. Versato Euro 99.650.000,00
Cod. Fisc. e Reg. Imprese di: Milano Monza Brianza Lodi n.11628710961
Partita IVA 11628710961
Sede legale: Circonvallazione Idroscalo s.n.c.20054 Segrate (MI) - Italia
***@***.***

File: markdwn.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+CatalogContent:
+  - 'learn-html'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+[**Markdown**](https://daringfireball.net/projects/markdown/) is a lightweight markup language for creating formatted text using a plain-text editor. It was created by John Gruber and [Aaron Swartz](https://www.codecademy.com/resources/docs/general/historical-technical-figures/aaron-swartz) in 2004 as a language that is appealing to human readers in its source code form.
+
+With Markdown, the display of the document is programmed by the user; formatting words as bold or italic, adding images and links, and creating lists and tables are just a few of the functionalities.
+
+Codecademy Docs e molti altri progetti memorizzano il loro README con un'estensione di file ".md", il che significa che il file è scritto in un linguaggio di markup chiamato Markdown. Utilizzando Markdown, gli scrittori possono specificare la formattazione del testo con caratteri come asterischi e trattini bassi, nonché quantità limitate di HTML (il linguaggio di programmazione utilizzato per i siti Web).
```
Comment (SaviDahegaonkar) [inline markdwn.md:14]:
```suggestion
Codecademy Docs e molti altri progetti memorizzano il loro README con un'estensione di file ".md", il che significa che il file è scritto in un linguaggio di markup chiamato Markdown. Utilizzando Markdown, gli scrittori possono specificare la formattazione del testo con caratteri come asterischi e trattini bassi, nonché quantità limitate di HTML (il linguaggio di markup utilizzato per la struttura delle pagine web).
```

File: markdwn.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+Ad esempio, il seguente codice markdown (senza spazi prima e dopo la parola/frase) creerebbe un elenco con del testo in grassetto in ogni elemento:
+
+- ** Apple **: Malus domestica
+- ** Banana **: Musa acuminata
+- ** Cherry **: Prunus cerasus
```
Comment (SaviDahegaonkar) [inline markdwn.md:20]:
```suggestion
- **Apple**: Malus domestica
- **Banana**: Musa acuminata
- **Cherry**: Prunus cerasus
```
In markdown there should not be spaces between the text and the asterisks.

**Review by SaviDahegaonkar — APPROVED** [review]

**Comment by mamtawardhani** [comment]
Closing this PR as: it does not close any issue, CLA is not signed, file path is incorrect, content is incorrect.
Thank you!

**Pull request closed without merging**